packages/terra-paginator/translations/en.json

```diff
@@ -4,6 +4,8 @@
   "Terra.paginator.first": "First",
   "Terra.paginator.last": "Last",
   "Terra.paginator.pageIndex": "Page {pageNumber}",
+  "Terra.paginator.pageSelected": "Page {pageNumber} selected",
```

Comment:
nit: Do we have a Jira logged to get translations for other languages?

Reply:
https://jira.cerner.com/browse/TRANS-14744